### PR TITLE
SSIX-2454: Support failover to another server without re-login

### PR DESCRIPTION
**Pull request by srybakov**
https://sample6.atlassian.net/browse/SSIX-2454

### CODE REVIEW

File: SpringSessionGrailsPlugin.groovy [hunk 1]
```diff
@@ -1,27 +1,19 @@
...
-import grails.plugin.springsession.store.mongo.config.MongoStoreSpringDataConfig
-import grails.plugin.springsession.store.redis.config.RedisStoreSessionConfig
 import grails.plugin.webxml.FilterManager
-import grails.util.Environment
-import org.codehaus.groovy.grails.commons.GrailsApplication
 import org.springframework.web.filter.DelegatingFilterProxy
 
 class SpringSessionGrailsPlugin {
-    def version = "1.2.1-BUILD_SNAPSHOT"
+    def version = "1.2.2"
```
Comment (paulsobolev) [inline SpringSessionGrailsPlugin.groovy]: 2.0 ?

Comment (srybakov) [inline SpringSessionGrailsPlugin.groovy]: Sorry, didn't understand why 2.0?

Comment (srybakov) [inline SpringSessionGrailsPlugin.groovy]: Do you mean to change the version set version to `2.0`?

Comment (srybakov) [inline SpringSessionGrailsPlugin.groovy]: Discussed to change to 1.3

**Pull request merged**